src/components/CategoricalDisplay.jsx

```diff
@@ -102,7 +103,7 @@ const CategoricalDisplay = ({
     return (
         <div className="cq-categorical-display">
             {renderMobileTitle()}
-            <div className="cq-lookup-filters">
+            <div className={`cq-lookup-filters ${hideLookupFilter ? 'hide' : ''}`}>
```

Comment:
`hide` could be rather misleading, because in desktop this shows up as well. Perhaps `scroll-down` is more appropriate?